src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -3,7 +3,14 @@
     <cd-card title="Details"
              i18n-title
              class="col-sm-3 px-3">
-             Text
+      <dl>
+        <dt>FSID</dt>
+        <dd>{{ detailsCardData.fsid }}</dd>
+        <dt>Orchestror</dt>
+        <dd>{{ detailsCardData.orchestrator }}</dd>
+        <dt>Ceph version</dt>
```

Comment:
I haven't tried it locally, but what about adding a margin to the bottom to increase the [spacing](https://getbootstrap.com/docs/4.3/utilities/spacing/)?
```suggestion
        <dt class="mt-1">Orchestror</dt>
        <dd>{{ detailsCardData.orchestrator }}</dd>
        <dt class="mt-1">Ceph version</dt>
```

Reply:
I think some kind of margin would be the way to go, but since we were trying to style as little as possible for the time being I dind't want to add it. In case in the future we want to change all card's styling or anything else that could be affected by it